src/buildercore/trop.py

```diff
@@ -546,6 +546,36 @@ def render_cloudfront(context, template, origin_hostname):
             SslSupportMethod='sni-only'
         )
     }
+    if context['cloudfront']['errors']:
+        props['Origins'].append(cloudfront.Origin(
+            DomainName=context['cloudfront']['errors']['domain'],
+            # TODO: constant
+            Id='errors',
+            # no advantage in using cloudfront.S3Origin for public buckets
+            CustomOriginConfig=cloudfront.CustomOrigin(
+                HTTPSPort=443,
+                OriginProtocolPolicy='https-only'
+            )
+        ))
+        props['CacheBehaviors'] = [
+            cloudfront.CacheBehavior(
+                TargetOriginId='errors',
+                ForwardedValues=cloudfront.ForwardedValues(
+                    QueryString=False
+                ),
+                PathPattern=context['cloudfront']['errors']['pattern'],
+                ViewerProtocolPolicy='allow-all',
+            ),
+        ]
+        all_codes = [400, 403, 404, 405, 414, 416, 500, 501, 502, 503, 504]
```

Comment:
Then we should only configure `502` according to the docs: http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/troubleshooting-response-errors.html
However, I've seen forum reports for 503 and 504, so maybe all three?